sqlgen/db.go

```diff
@@ -174,7 +174,9 @@ func (db *DB) checkFilterAgainstLimits(ctx context.Context, filter Filter, table
 		limitFilter := db.dynamicLimit.GetLimitFilter(ctx, table.Name)
 		if limitFilter != nil {
 			if err := db.checkFilterAgainstLimit(filter, limitFilter); err != nil {
-				if keepGoing := db.dynamicLimit.ShouldContinueOnError(err, table.Name); !keepGoing {
+				clause, args := query.ToSQL()
+				detailedErr := fmt.Errorf("%s; clause: '%s'; args: '%v'", err.Error(), clause, args)
```

Comment:
consider creating a new error type with more information that wraps the previous error and conforms to the go1.13 error wrapping.

https://ieftimov.com/post/deep-dive-in-upcoming-go-error-inspection-changes/

Reply:
Hmm, interesting.  I'll add this to a new `errors.go` in the `sqlgen` package.

Reply:
Done.